jscl.lisp

```diff
@@ -58,6 +58,7 @@
 ;;; compiled in the host
 (defvar *source*
   '(("boot"          :target)
+    ("early-char"    :target)
```

Comment:
I wasn't 100% sure why some were in boot, and then redefined in char. My guess was that, the ones in boot were needed earlier-on, so rather than having two different implementations, I consolidated them here. Perhaps I was too cautious.